connectors/CompaniesHouse/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+{
+  "basePath": "/",
+  "consumes": [],
+  "definitions": {},
+  "host": "api.companieshouse.gov.uk",
+  "info": {
+    "description": "UK Company Search",
+    "title": "Companies House",
+    "version": "1.0"
+  },
+  "parameters": {},
+  "paths": {
+    "/api.companieshouse.gov.uk/search/companies": {},
+    "/search/companies": {
+      "get": {
+        "description": "Search Company",
+        "operationId": "SearchCompany",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "q",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "default": {
```

Comment:
Please use expected response code instead of the 'default'

Reply:
Updated to '200' response and added a default.